src/util/tags.ts

```diff
@@ -42,6 +42,7 @@ function tagify(
         return null;
     }
 
+    enforceWhitelist ??= true;
```

Comment:
Why not up top?

Reply:
No particular reason, just seemed pretty. I can move it.